controllers/useraccount/useraccount_controller.go

```diff
@@ -216,12 +217,29 @@ func (r *Reconciler) ensureUser(logger logr.Logger, config membercfg.Configurati
 	logger.Info("user already exists", "name", userAcc.Name)
 
 	// ensure mapping
-	if user.Identities == nil || len(user.Identities) < 1 || user.Identities[0] != ToIdentityName(userAcc.Spec.UserID, config.Auth().Idp()) {
+	expectedIdentities := []string{ToIdentityName(userAcc.Spec.UserID, config.Auth().Idp())}
+	if userAcc.Spec.OriginalSub != "" {
+		expectedIdentities = append(expectedIdentities, ToIdentityName(fmt.Sprintf("b64:%s", base64.RawStdEncoding.EncodeToString([]byte(userAcc.Spec.OriginalSub))), config.Auth().Idp()))
```

Comment:
it would be nice to have and use this as a function:
```
ToIdentityName(fmt.Sprintf("b64:%s", base64.RawStdEncoding.EncodeToString([]byte(userAcc.Spec.OriginalSub))), config.Auth().Idp())
``` 
for example `ToIdentityInBase64(userAcc.Spec.OriginalSub, config.Auth().Idp())` so we are sure, that we use always the same format here as well as in the other lines
and it's also better to read.

Reply:
We can introduce this once we remove all the migration code.

Reply:
Ok, then just please keep that in mind :pray: